### DISCUSSION TITLE
Crops doesn't update after component load

### DISCUSSION THREAD

**Discussion by ZeroRetro** [berviantoleo/react-multi-crop#1823] [Q&A]
Imagine that we store the images with the crops we previously did. When loading that image again the `crops` comes as an `empty array` and after a request it comes with the real value. 

The component is not being updated and stay with the first value it receive. 

Maybe some `useEffect` for the `record` parameter will be enough.

The crops array:
![image](https://user-images.githubusercontent.com/33663801/236810647-1016a134-d7ce-4d7a-9177-b8379622f9f6.png)

New crops are working perfect, and I can force the component to not render if the crops array is empty, but this is just a workaround because if the image really doesn't have any crop then it will never be rendered.

**Comment by berviantoleo** [comment]
Yes, currently, the components expect the record as the read-only props. The component expects the parent to store the updated records from `onChange` and give the updated value to the records if the parent wants to have the latest crops.

I believe we shouldn't change the value of props.

**Reply by ZeroRetro** [reply]
I managed to solve my issue by changing the `key` and forcing the component to render again
